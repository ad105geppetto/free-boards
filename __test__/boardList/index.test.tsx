import "@testing-library/jest-dom";
import { act, render } from "@testing-library/react";
import BoardListUI from "../../src/components/units/board/list/BoardList.presenter";
import { propsMock } from "./index.mock";

describe("자유게시판 목록 페이지 Unit Test", () => {
  it("게시판 목록 페이지 스냅샷 테스트", async () => {
    await act(async () => {
      const result = render(<BoardListUI {...propsMock} />);

      expect(result.container).toMatchSnapshot();
    });
  });
});
