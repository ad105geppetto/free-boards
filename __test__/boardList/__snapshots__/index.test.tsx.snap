// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`자유게시판 목록 페이지 Unit Test 게시판 목록 페이지 스냅샷 테스트 1`] = `
<div>
  <div
    class="css-1ol1jm3"
  >
    <section
      class="css-ztv6jj"
    >
      <h3
        class="css-pd6ube"
      >
        자유게시판
      </h3>
      <div
        class="css-1hstsbv"
      >
        <div
          class="css-vmu8p2"
        >
          <div
            class="css-1kfaxq7"
          >
            번호
          </div>
          <div
            class="css-1paygf6"
          >
            제목
          </div>
          <div
            class="css-yd5q4q"
          >
            작성자
          </div>
        </div>
        <a
          class="css-x243er"
          href="/free-boards/ih2dfdf2"
        >
          <div
            class="css-1kfaxq7"
          >
            5
          </div>
          <div
            class="css-1paygf6"
          >
            한글오라와와와와와와와와와와와와와와와와
          </div>
          <div
            class="css-yd5q4q"
          >
            한글이름입니다앙
          </div>
        </a>
        <a
          class="css-x243er"
          href="/free-boards/nrfdhd89"
        >
          <div
            class="css-1kfaxq7"
          >
            4
          </div>
          <div
            class="css-1paygf6"
          >
            한글오라와와와와와와와와와와와와와와와와
          </div>
          <div
            class="css-yd5q4q"
          >
            한글이름입니다앙
          </div>
        </a>
        <a
          class="css-x243er"
          href="/free-boards/hkjjhkec4t"
        >
          <div
            class="css-1kfaxq7"
          >
            3
          </div>
          <div
            class="css-1paygf6"
          >
            한글오라와와와와와와와와와와와와와와와와
          </div>
          <div
            class="css-yd5q4q"
          >
            한글이름입니다앙
          </div>
        </a>
        <a
          class="css-x243er"
          href="/free-boards/sdfdsfsjhhj"
        >
          <div
            class="css-1kfaxq7"
          >
            2
          </div>
          <div
            class="css-1paygf6"
          >
            한글오라와와와와와와와와와와와와와와와와
          </div>
          <div
            class="css-yd5q4q"
          >
            한글이름입니다앙
          </div>
        </a>
        <a
          class="css-x243er"
          href="/free-boards/ikfnsidfods"
        >
          <div
            class="css-1kfaxq7"
          >
            1
          </div>
          <div
            class="css-1paygf6"
          >
            한글오라와와와와와와와와와와와와와와와와
          </div>
          <div
            class="css-yd5q4q"
          >
            한글이름입니다앙
          </div>
        </a>
      </div>
      <div
        class="css-1e9voc2"
      >
        <a
          class="css-1kb8w1a"
          href="/free-boards/new"
        >
          게시글 작성
        </a>
      </div>
      <div
        class="css-1yqh4in"
      >
        <button
          class="css-1eaf97t"
        >
          이전 페이지
        </button>
        <button
          class="css-1eaf97t"
        >
          다음 페이지
        </button>
      </div>
    </section>
  </div>
</div>
`;
